extensions/xcode/src/open-url-in-simulator.command.tsx

```diff
@@ -0,0 +1,37 @@
+import { XcodeSimulatorService } from "./services/xcode-simulator.service";
+import { Clipboard, closeMainWindow, showToast, Toast } from "@raycast/api";
+
+interface OpenSimulatorArguments {
+  url?: string;
+}
+
+export default async (props: { arguments: OpenSimulatorArguments }) => {
+  const toast = await showToast({
+    style: Toast.Style.Animated,
+    title: "Opening URL in Simulator...",
+  });
+  try {
+    const url = props.arguments.url ?? (await Clipboard.readText());
+    if (!url) throw Error("Please provide or copy a valid URL.");
+    const result = await XcodeSimulatorService.openUrl(url);
+    toast.style = Toast.Style.Success;
+    toast.title = "URL opened in Simulator.";
+    XcodeSimulatorService.showSimulator();
+    await new Promise((r) => setTimeout(r, 1500));
```

Comment:
Is this timeout mandatory?

Reply:
It's here to avoid dismissing the Raycast window right away without having the success message, so might not be mandatory since you have a direct feedback with the simulator showing up